lavalink/WebSocket.py

```diff
@@ -95,6 +108,10 @@ def __init__(self, lavalink, host, password, ws_port, ws_retry, shard_count):
                 data = json.loads(await self._ws.recv())
             except websockets.ConnectionClosed as error:
                 log.warning('Disconnected from Lavalink %s', str(error))
+                for g, p in self._lavalink.players:

```

Comment:
Would there be any benefit from swapping this over to `.keys()` as the `p` variable is unused?

Reply:
There isn't any performance difference. So it would be just fine swapping it to `for g in self._lavalink.players._players.keys()`